src/app/components/AudioVideoHead/index.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import Helmet from 'react-helmet';
+import { arrayOf, shape, string } from 'prop-types';
+
+const AudioVideoHead = ({ audioVideoAssets }) => {
+  const settingsArray = audioVideoAssets.map(
+    ({ mediaPlayerSettings }) => mediaPlayerSettings,
+  );
+
+  // The [${settingsArray}] syntax is needed due to its usage in the string literal
+  const loadPlayers = `
+      var settings = [${settingsArray}];
+      ${audioVideoAssets
+        .map(
+          (avAsset, index) =>
+            `var player${index} = bump.player(document.getElementById('${avAsset.id}'), settings[${index}]);
+          player${index}.load();`,
+        )
+        .join(' ')}
+    `;
+
+  return (
+    <>
+      <Helmet>
+        <script type="text/javascript">
+          {`
+            function mediaPlayerSetup(container) {
+              require(['bump-4'], (bump) => {
+                ${loadPlayers}
+              });
+            }
+          `}
+        </script>
+        <script type="text/javascript">
+          {`
+            function initialiseRequires() {
+              var requiredScripts = { "bump-4": "https://emp.bbci.co.uk/emp/bump-4/bump-4" };
+              require({ paths: requiredScripts, waitSeconds: 30 });
+              mediaPlayerSetup();
+            }
+          `}
+        </script>
+        <script
+          onLoad="initialiseRequires()"
+          type="text/javascript"
+          src="https://static.bbci.co.uk/frameworks/requirejs/0.13.0/sharedmodules/require.js"
+        />
+      </Helmet>
+    </>
+  );
+};
+
+AudioVideoHead.propTypes = {
+  audioVideoAssets: arrayOf(
+    shape({
+      id: string,
```

Comment:
Is this not required, and if not, what is its default assignment?

Reply:
This will need refactoring as part of https://github.com/bbc/simorgh/issues/2307 so we can pick up then.